pretty.go

```diff
@@ -0,0 +1,97 @@
+package microerror
+
+import (
+	"errors"
+	"strings"
+	"unicode"
+)
+
+const (
+	prefix = "Error: "
+	suffix = " error"
+)
+
+func Pretty(err error) string {
```

Comment:
How about making it `func Pretty(err error, stack bool)` for optional stack printing?

Reply:
I can do that 👍